### PR TITLE
TDL-9902 removed the automatic fields from ad ext report

### PR DESCRIPTION
**Pull request by namrata270998**
# Description of change
Removed the constraint imposed on AdExtensionDetailReport of having 'Ctrl', 'Clicks', and 'Impressions' from the required fields in the reports.py

# Manual QA steps
 - Check that the automatic fields test runs without any failures
 - Check that the catalog generated has available inclusion property for Ctr, Clicks, and Impression fields for AdExtensionDetailReport 
 
# Risks
 - 
 
# Rollback steps
 - revert this branch

### CODE REVIEW

**Review by KrisPersonal — APPROVED** [review]

**Review by KrisPersonal — APPROVED** [review]

File: tap_bing_ads/reports.py [hunk 1]
```diff
@@ -27,10 +27,7 @@
         'AdExtensionId',
         'AdExtensionPropertyValue',
         'AdExtensionType',
-        'AdExtensionTypeId',
-        'Clicks',
-        'Ctr',
-        'Impressions'
+        'AdExtensionTypeId'
```
Comment (karanpanchal-crest) [inline tap_bing_ads/reports.py:30]: @namrata270998 Can you please add a comment regarding the change you have made in this PR.
Also add a code comment above this line regarding what these fields are meant for

**Review by karanpanchal-crest — CHANGES_REQUESTED** [review]

**Comment by karanpanchal-crest** [comment]
@umangagrawal-crest @namrata270998 As discussed with Krishnan, please add a comment in the JIRA regarding what needs to be added to the documentation or the behavioral change incorporated as part of this PR

File: tap_bing_ads/reports.py (hunk 1 above)
Comment (namrata270998) [reply tap_bing_ads/reports.py:30]: Bing has a requirement that each report must have at least 1 performance based metric (Clicks, Impressions, etc.) selected to generate a report. We force `Impressions`, `Ctr,`, and `Clicks` to be automatic for the `ad_extension_detail_report`. This is inconsistent with how we treat field selection for other reports as we do not impose any performance measures on any other report streams. It should be up to user, which perf measures should be selected. Hence, removed the `Impressions`, `Ctr,`, and `Clicks` from required fields.

**Review by karanpanchal-crest — APPROVED** [review]

**Review by kspeer825 — CHANGES_REQUESTED** [review]
While the base expectations were properly updated, we are not actually testing this feature. The assertions are commented out in the discovery test, because the tap does not conform to the metadata standards. I believe this PR must incorporate the changes from https://github.com/singer-io/tap-bing-ads/pull/91

**Comment by namrata270998** [comment]
> 

@kspeer825 Sure will merge this PR after the [#91](https://github.com/singer-io/tap-bing-ads/pull/91) gets merged.

**Review by kspeer825 — CHANGES_REQUESTED** [review]
I can approve this when the dependencies are brought in from #91. I am not able to keep up with the changes across PRs to reconcile the differences. Ya'll are too fast for me with these PRs. In the future we will try to organize the Jira tickets so that there is a clear order of operations and one ticket will not unexpectedly block another after the fact.

**Comment by namrata270998** [comment]
> I can approve this when the dependencies are brought in from #91. I am not able to keep up with the changes across PRs to reconcile the differences. Ya'll are too fast for me with these PRs. In the future we will try to organize the Jira tickets so that there is a clear order of operations and one ticket will not unexpectedly block another after the fact.

@kspeer825 we merged the code of PR 91 here and the build is passing, can you please review it now?

**Review by kspeer825 — APPROVED** [review]

**Pull request merged**